pkg/cmd/init/scenario/init/bootstrap_cluster_role.yaml

```diff
@@ -34,4 +34,9 @@ rules:
   - "managedclustersets/join"
   verbs: 
   - "create"
-  
\ No newline at end of file
+- apiGroups:
```

Comment:
I do no think agent needs this clusterrole?

Reply:
the auto approval feature is controlled by the clustermanager, when we use `clusteradm join` a cluster, we need read the clustermanager to see it if enable the auto approval feature, so I added  this clusterrole for bootstrap token

Reply:
But it is mainly for clusteradm to know, not agent, right?

Reply:
right, it is mainly for clusteradm to know

Reply:
for clusteradm join, it seems we only have the bootstrap token, right?

Reply:
or we give a clear messages instead of checking the auto approval feature? e.g.

```
Your managed cluster cluster1 has joined the Hub.

If the ManagedClusterAutoApproval feature is not enabled, Please log onto the hub ...
```

Reply:
yes, I think this would be better.